package.json

```diff
@@ -15,7 +16,8 @@
     "eslint-plugin-prettier": "^3.0.1",
     "lint-staged": "^8.1.5",
     "pre-commit": "^1.2.2",
-    "prettier": "^1.17.0"
+    "prettier": "^1.17.0",
+    "sketch-app-types": "^0.1.1"
```

Comment:
if you reference it from the types declaration, it needs to be in `dependencies`

But is it needed? I had a quick look at the types and couldn't find a reference to it?

Reply:
`sketch-app-types` is a basic type definition module for `sketch` object. `BrowserWindowOptions` has `parent` property which type is sketch Document. the Document definition depends on `sketch/dom`, so I add `sketch-app-types` dependency for our module.
I think the definition module just need in development environment, so I add it to devdependencies. It's not a big problem, I'll change it~